1config-core/src/com/brunobonacci/oneconfig/util.clj

```diff
@@ -235,15 +235,27 @@
   (or (env "HOME") (env "userprofile")))
 
 
+(defn dir-exists?
+  [path]
+  (when-let [dir (some-> path io/file)]
+    (and (.exists ^File dir) (.isDirectory ^File dir))))
+
 
 (defn home-1config
-  "returns ~/.1config/"
+  "if ONECONFIG_HOME is set in the environment, it returns
+  <ONECONFIG_HOME>/.1config, otherwise it returns ~/.1config"
   []
```

Comment:
I think that the behaviour of this function should be more like if `ONECONFIG_HOME` then it is returned,
if it isn't defined then it does what it was doing before like: `$HOME/`+`.1config`
so it makes it clear that when you define your `export ONECONFIG_HOME=/foo`, `/foo` is going to be used as 1Config home and if not defined by default is : `$HOME/`+`.1config`